src/mixxx.h

```diff
@@ -171,6 +174,7 @@ class MixxxMainWindow : public QMainWindow {
 
     void linkSkinWidget(ControlObjectSlave** pCOS,
                         ConfigKey key, const char* slot);
+    QString getTitleFromTrack(TrackPointer pTrack);
```

Comment:
please remove this function and the includes for it. You are never using it. I'm actually a little bit puzzled why this compiles at all.